### PR TITLE
Update storage-costs.mdx

### PR DESCRIPTION
**Pull request by litchipi**
Related to https://github.com/massalabs/massa/issues/4371

Divide storage costs by 10

### CODE REVIEW

File: docs/learn/storage-costs.mdx [hunk 1]
```diff
@@ -6,19 +6,19 @@ sidebar_label: Storage costs
...
 This limit is essential to encourage widespread adoption and enable users to run nodes from their homes.
 To ensure compliance with this limit, a mechanism has been established that correlates storage space with locked coins.
 
 For every byte of storage space claimed, whether it's for storing address and balance information, keys in the datastore, bytecode, or other data elements,
  users are required to lock a corresponding amount of coins.
 These locked coins act as a commitment and are released when the allocated storage space is released.
 
-To determine the amount of coins needed to lock for each byte of storage, the value of 0.00025 Massa coin (MAS) has been chosen.
-This value ensures that if half of the total coins (250,000,000 MAS) are locked, the storage limit of 1TB will be reached.
+To determine the amount of coins needed to lock for each byte of storage, the value of 0.000025 Massa coin (MAS) has been chosen.
```
Comment (damip) [inline docs/learn/storage-costs.mdx:17]:
```suggestion
To determine the amount of coins needed to lock for each byte of storage, the value of 0.0001 Massa coin (MAS) has been chosen.
```

Comment (damip) [reply docs/learn/storage-costs.mdx:17]: (yes the values in the doc were wrong)

File: docs/learn/storage-costs.mdx [hunk 2]
```diff
@@ -6,19 +6,19 @@ sidebar_label: Storage costs
...
+This value ensures that if half of the total coins (250,000,000 MAS) are locked, the storage limit of 10TB will be reached.
 
 To avoid confusion, a fixed size of 8 bytes has been chosen for each balance.
-Therefore, the initial ledger entry for address and balance incurs a cost of (8 + 32) * 0.00025 = 0.01 MAS.
+Therefore, the initial ledger entry for address and balance incurs a cost of (8 + 32) * 0.000025 = 0.001 MAS.
```
Comment (damip) [inline docs/learn/storage-costs.mdx:21]:
```suggestion
The initial ledger entry for address and balance incurs a cost of 0.001 MAS.
```

File: docs/learn/storage-costs.mdx [hunk 3]
```diff
@@ -6,19 +6,19 @@ sidebar_label: Storage costs
...
 For every byte of storage space claimed, whether it's for storing address and balance information, keys in the datastore, bytecode, or other data elements,
  users are required to lock a corresponding amount of coins.
 These locked coins act as a commitment and are released when the allocated storage space is released.
 
-To determine the amount of coins needed to lock for each byte of storage, the value of 0.00025 Massa coin (MAS) has been chosen.
-This value ensures that if half of the total coins (250,000,000 MAS) are locked, the storage limit of 1TB will be reached.
+To determine the amount of coins needed to lock for each byte of storage, the value of 0.000025 Massa coin (MAS) has been chosen.
+This value ensures that if half of the total coins (250,000,000 MAS) are locked, the storage limit of 10TB will be reached.
 
 To avoid confusion, a fixed size of 8 bytes has been chosen for each balance.
```
Comment (damip) [inline docs/learn/storage-costs.mdx:20]: This can be removed:

```suggestion```

File: docs/learn/storage-costs.mdx [hunk 4]
```diff
@@ -27,7 +26,7 @@ and the sum of constants for each datastore key and its corresponding value size
 The overall formula is:
 
 $$
-Cost_{massa} = 0.00025 * (AddressSize + BalanceSize + BytecodeSize + \sum_i (DataStoreKey_i + DatastoreValue_i))
+Cost_{massa} = 0.000025 * (AddressSize + BalanceSize + BytecodeSize + \sum_i (DataStoreKey_i + DatastoreValue_i))
```
Comment (damip) [inline docs/learn/storage-costs.mdx:29]: update this

File: docs/learn/storage-costs.mdx [hunk 5]
```diff
@@ -36,27 +35,27 @@ the associated storage costs will be charged to the address making the call.
 
 
 ## Examples
-When creating an address on the blockchain, a minimum of 0.01 MAS (0.00025 * (32 + 8)) needs to be sent alongside the operation that generates the address (e.g., a transfer).
+When creating an address on the blockchain, a minimum of 0.001 MAS (0.000025 * (32 + 8)) needs to be sent alongside the operation that generates the address (e.g., a transfer).
```
Comment (damip) [inline docs/learn/storage-costs.mdx:38]: update this

File: docs/learn/storage-costs.mdx [hunk 6]
```diff
@@ -36,27 +35,27 @@ the associated storage costs will be charged to the address making the call.
...
 This cost covers the creation of the address and its associated balance.
 
 
 Suppose you want to store your birth date, which is 30 bytes long, in your datastore.
 In that case, you need to send an operation that creates a key in your datastore using a Smart Contract.
-This operation incurs a storage cost of at least 0.01 MAS, calculated as 0.00025 * (10 + 30).
+This operation incurs a storage cost of at least 0.001 MAS, calculated as 0.000025 * (10 + 30).
```
Comment (damip) [inline docs/learn/storage-costs.mdx:44]: update this

File: docs/learn/storage-costs.mdx [hunk 7]
```diff
@@ -36,27 +35,27 @@ the associated storage costs will be charged to the address making the call.
...
+However, until a transfer of at least 0.001 Massa is received to your wallet address, it will not appear in any of blockchain's ledgers.
 
 If the amount of transfer to an unexisting user address is lower than the amount of MAS coin needed to create the address on the ledger, the operation will be refused.
 
 
 :::tip
 1. If a transfer of 0.005 MAS is received, the operation will fail.
 
-2. If a transfer of 10 MAS is received, the balance of the newly created address will show 9.99 MAS, as 0.01 MAS represent the storage cost.
+2. If a transfer of 10 MAS is received, the balance of the newly created address will show 9.999 MAS, as 0.001 MAS represent the storage cost.
```
Comment (damip) [inline docs/learn/storage-costs.mdx:55]: update this

File: docs/learn/storage-costs.mdx [hunk 8]
```diff
@@ -36,27 +35,27 @@ the associated storage costs will be charged to the address making the call.
...
 :::
 
-In case you wish to delete the entry in your datastore, you will be refunded the storage costs (0.01 MAS).
+In case you wish to delete the entry in your datastore, you will be refunded the storage costs (0.001 MAS).
```
Comment (damip) [inline docs/learn/storage-costs.mdx:58]:
```suggestion
In case you wish to delete the entry in your datastore, you will be refunded the storage costs.
```

**Review by damip — COMMENTED** [review]
LGTM but let's wait for buildnet update to merge it to the docs

**Comment by litchipi** [comment]
Let's merge once https://github.com/massalabs/massa-sc-toolkit/pull/355 is merged

**Comment by damip** [comment]
I updated the document

**Pull request merged**